README.md

```diff
@@ -1,23 +1,176 @@
+        - [scikit-learn](http://scikit-learn.org/stable/install.html) 0.17\*\*
+    - spark.ml client:
+        - Java 1.8+
+        - Spark 2.0.0\*\*
+
+    **(\*\*) = Must have exact version**
+
+    On OSX, we also assume that you have [homebrew](https://brew.sh/) installed.
+
+    **On Mac OSX:**
+
+    ```bash
+    # Use homebrew to install the dependencies
+    brew install sqlite
+    brew install maven
+    brew install node
+    brew install sbt # for spark.ml client only
+    brew install mongodb
+
+    # ModelDB works only with Thrift 0.9.3 and 0.10.0. If you do not have thrift installed, install via brew.
+
+    brew install thrift
+    ```
+
+    **On Linux:**
+
+    ```bash
+    apt-get update
+    sudo apt-get install sqlite
+    sudo apt-get install maven
+    sudo apt-get install sbt # for spark.ml client only
+    sudo apt-get install nodejs # may need to symlink node to nodejs. "cd /usr/bin; ln nodejs node"
+    sudo apt-get install -y mongodb-org # further instructions here: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/
+
+    # install thrift. [path_to_thrift] is the installation directory
+    cd [path_to_thrift]
+    wget http://mirror.cc.columbia.edu/pub/software/apache/thrift/0.9.3/thrift-0.9.3.tar.gz
+    tar -xvzf thrift-0.9.3.tar.gz
+    cd thrift-0.9.3
+    ./configure
+    make
+    export PATH=[path_to_thrift]/:$PATH
+    ```
+
+    For Linux, you can also refer to [this script](https://github.com/mitdbg/modeldb/blob/master/docs/install_on_linux.sh). 
+
+3. **Build**
+    
+    ModelDB is composed of three components: the ModelDB **server**, the ModelDB **client libraries**, and the ModelDB **frontend**.
+
+    In the following, **[path_to_modeldb]** refers to the directory into which you have cloned the modeldb repo and **[thrift_version]** is 0.9.3 or 0.10.0 depending on your thrift version (check by running ```thrift -version```).
+
+    ```bash
+    # run the script to set up the sqlite and the mongodb databases that modeldb will use
+    # note that this clears any previous modeldb databases
+    # this also starts mongodb
+    cd [path_to_modeldb]/server/codegen
+    ./gen_sqlite.sh
+
+    # build and start the server
+    cd ..
+    ./start_server.sh [thrift_version] &
+
+    # build scikit-learn client library
+    cd [path_to_modeldb]/client/python
+    pip install -r requirements.txt
+    ./build_client.sh
+
+    # build spark.ml client library
+    cd [path_to_modeldb]/client/scala/libs/spark.ml
+    ./build_client.sh
+
+    # start the frontend
+    cd [path_to_modeldb]/frontend
+    ./start_frontend.sh & # the frontend will now be available in http://localhost:3000/
+
+    # shutdown mongodb server after killing the server
+    # on Mac OSX and Linux
+    mongo --eval "db.getSiblingDB('admin').shutdownServer()" 
+    ```
+
+## Usage and Samples
+
+### Incorporate ModelDB into your ML workflow
+- [scikit-learn](client/python)
+- [spark.ml](client/scala/libs/spark.ml)
```

Comment:
Can you link to https://github.com/mitdbg/modeldb/blob/master/docs/getting_started/spark_ml.md?

Also please remove the installation instructions on that page and only keep the workflow ones